### ISSUE TITLE
Rubocop has dropped support for Ruby 2.3

### ISSUE THREAD

**Issue by chrisseaton** [Shopify/statsd-instrument#263]
Rubocop has dropped support for Ruby 2.3:

https://github.com/rubocop-hq/rubocop/commit/58738943eb3f7a41266b4546a829fd0a5a13703d

Which we use in tests:

https://github.com/Shopify/statsd-instrument/blob/d1083b638bf0f3b225edfd035b61e17848d56cf7/test/helpers/rubocop_helper.rb#L11

I don't think this gem pins the version of Rubocop in any way - unless I'm missing something? So these tests now fail.

```
  1) Error:
Rubocop::SplatArgumentsTest#test_no_offenses:
ArgumentError: RuboCop found unknown Ruby version: 2.3
    /Users/chrisseaton/.gem/ruby/2.6.5/gems/rubocop-0.82.0/lib/rubocop/processed_source.rb:191:in `parser_class'
    /Users/chrisseaton/.gem/ruby/2.6.5/gems/rubocop-0.82.0/lib/rubocop/processed_source.rb:200:in `create_parser'
    /Users/chrisseaton/.gem/ruby/2.6.5/gems/rubocop-0.82.0/lib/rubocop/processed_source.rb:158:in `parse'
    /Users/chrisseaton/.gem/ruby/2.6.5/gems/rubocop-0.82.0/lib/rubocop/processed_source.rb:36:in `initialize'
    /Users/chrisseaton/src/github.com/Shopify/statsd-instrument/test/helpers/rubocop_helper.rb:11:in `new'
    /Users/chrisseaton/src/github.com/Shopify/statsd-instrument/test/helpers/rubocop_helper.rb:11:in `assert_no_offenses'
    /Users/chrisseaton/src/github.com/Shopify/statsd-instrument/test/rubocop/splat_arguments_test.rb:15:in `test_no_offenses'
```

I found this problem in TruffleRuby, but seems to apply to MRI as well.

**Comment by wvanbergen** [comment]
Let me see if I can fix this.

**Comment by wvanbergen** [comment]
Fixed by #264.
